tests/test_create.py

```diff
@@ -2548,3 +2549,32 @@ def test_remove_ignore_nonenv(tmp_path: Path, conda_cli: CondaCLIFixture):
 
     assert filename.exists()
     assert tmp_path.exists()
+
+
+def test_repodata_v2_base_url(
+    tmp_path: Path,
+    conda_cli: CondaCLIFixture,
+    monkeypatch: MonkeyPatch,
+):
+    if context.solver == "libmamba":
+        pytest.skip("conda-libmamba-solver doesn't implement base_url support yet")
```

Comment:
Can we verify this somehow programmatically?

Reply:
We can xfail it if you prefer, but I don't think libmamba exposes a symbol anywhere with the max supported repodata version ([see relevant function in v2](https://github.com/jaimergp/mamba/blob/3a31b8390ab7f84c4768e73c83f8b92980877afe/libmamba/src/solver/libsolv/helpers.cpp#L474)). I also [asked](https://matrix.to/#/!mTrfGUswppBVkBaxMn:gitter.im/$P3Kf773XvSzJKc20SB-E59WME3D1fBtQ2bYeO-3KEKI?via=gitter.im&via=matrix.org) in their Matrix channel and they confirmed it's not supported.

Reply:
See [81ed09c](https://github.com/conda/conda/pull/13744/commits/81ed09c2d284a42362f1de6c579c3e8b8e4d8717)